.github/workflows/differential-shellcheck.yml

```diff
@@ -0,0 +1,17 @@
+name: Differential ShellCheck
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+
+    steps: 
+      - name: Repository checkout
+        uses: actions/checkout@v2
+
+      - name: Differential ShellCheck
+        uses: redhat-plumbers-in-action/differential-shellcheck@v1
```

Comment:
Ah, I figured it out- 
Seems I forgot to fetch latest commits .... 

```
remote: Enumerating objects: 5, done.
remote: Counting objects: 100% (5/5), done.
remote: Total 5 (delta 4), reused 4 (delta 4), pack-reused 0
Unpacking objects: 100% (5/5), 582 bytes | 291.00 KiB/s, done.
From github.com:redhat-plumbers/dracut-rhel9
   1cd36263..32524aa0  main       -> plumbers-9/main

```

Reply:
Which is actually just small change which was merged in the meantime:

```diff
diff --git a/.github/workflows/differential-shellcheck.yml b/.github/workflows/differential-shellcheck.yml
index 095c3fe6..55273b3d 100644
--- a/.github/workflows/differential-shellcheck.yml
+++ b/.github/workflows/differential-shellcheck.yml
@@ -1,7 +1,5 @@
 name: Differential ShellCheck
 on:
-  push:
-    branches: [main]
   pull_request:
     branches: [main]

@@ -11,7 +9,11 @@ jobs:

     steps:
       - name: Repository checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0

       - name: Differential ShellCheck
-        uses: redhat-plumbers-in-action/differential-shellcheck@v1
+        uses: redhat-plumbers-in-action/differential-shellcheck@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}

```

Reply:
On that topic, do we need `b/.github/labeler.yml` ?

Also this change I'm not sure what to do about:
https://gist.github.com/afedcdea1b691f90e7bca56f0aa676d3
(most probably comes from upstream)

Reply:
@jamacku WDYT?

Reply:
What do you think? (sorry; abbrev)

But I guess current state (as long as it works), is ok.... What I was asking is about

 - `.github/labeler.yml` - as I've noticed there were some downstream changes previously (I don't remember from the top of my head; probably removed)
 - https://gist.github.com/afedcdea1b691f90e7bca56f0aa676d3
   - IOW: `.github/workflows/integration.yml` and
   - `.github/workflows/container.yml`

As I have done some manual changes, due to conflicts with upstream (or what did make sense for me). 

I hope those files are OK :]

Reply:
The Fedora stuff was taken from the upstream directly, I added only the C8S stuff, so bumping Fedora to F34/F35 would definitely make sense.

Reply:
I've synced (some of) the changes from upstream (there's no longer 34/33, but just 'latest'), along with the newly created files.

Reply:
Thanks for checking!